Yoast/YoastSEO.js
# Add seo analysis to example and display results

## Summary

This PR can be summarized in the following changelog entry:

* Adds SEO assessments to the webworker
* Returns readability and SEO assessment results to the wrapper

## Test instructions

This PR can be tested by following these steps:

* Run the example and check if the SEO assessments appear
* Change input (text, keyword, synonyms) and check if readability and SEO assessment results have changed.

Fixes #1620
